frontend/src/utils/game/createGameObject.js

```diff
@@ -4,46 +4,217 @@ function createGameObject(scene) {
-  ballPlaneMaterial.side = THREE.DoubleSide;
-  const ballPlane = new THREE.Mesh(ballPlaneGeometry, ballPlaneMaterial);
-  ballPlane.rotation.y = Math.PI / 2;
+  const ballPlanes = [
+    {
+      w: lineWidth,
+      h: planeY + lineWidth,
+      d: lineWidth,
+      position: { x: 0, y: 0, z: planeZ / 2 },
+      rotation: { x: 0, y: 0, z: 0 },
+    },
+    {
+      w: lineWidth,
+      h: planeY + lineWidth,
+      d: lineWidth,
+      position: { x: 0, y: 0, z: -planeZ / 2 },
+      rotation: { x: 0, y: 0, z: 0 },
+    },
+    {
+      w: lineWidth,
+      h: lineWidth,
+      d: planeZ + lineWidth,
+      position: { x: 0, y: planeY / 2, z: 0 },
+      rotation: { x: 0, y: 0, z: 0 },
+    },
+    {
+      w: lineWidth,
+      h: lineWidth,
+      d: planeZ + lineWidth,
+      position: { x: 0, y: -planeY / 2, z: 0 },
+      rotation: { x: 0, y: 0, z: 0 },
+    },
+  ];
+
+  ballPlanes.forEach((plane) => {
+    const mesh = new THREE.Mesh(
+      new THREE.BoxGeometry(plane.w, plane.h, plane.d),
+      ballPlaneMaterial
+    );
+    mesh.position.set(plane.position.x, plane.position.y, plane.position.z);
+    mesh.rotation.set(plane.rotation.x, plane.rotation.y, plane.rotation.z);
+    ballPlane.add(mesh);
+  });
+
   ballPlane.position.set(0, 0, 0);
   ballPlane.name = 'ballPlane';
   scene.getObjectByName('map').add(ballPlane);
 
-  const redPaddleGeometry = new THREE.BoxGeometry(0.5, 3, 3);
-  const redPaddleMaterial = new THREE.MeshBasicMaterial({
-    color: 0xff0000,
+  // paddle
+  const paddleWidth = 0.5;
+  const paddleHeight = 3;
+  const paddleDepth = paddleHeight * 1.5;
+
+  const paddleGeometry = new THREE.BoxGeometry(
+    paddleWidth,
+    paddleHeight,
+    paddleDepth
+  );
+  const paddleMaterial = new THREE.MeshBasicMaterial({
+    color: 0xa0a0a0,
     transparent: true,
-    opacity: 0.7,
+    opacity: 0.5,
   });
-  const redPaddle = new THREE.Mesh(redPaddleGeometry, redPaddleMaterial);
-  redPaddle.position.set(25, 0, 0);
+  const redPaddle = new THREE.Mesh(paddleGeometry, paddleMaterial);
+  const bluePaddle = new THREE.Mesh(paddleGeometry, paddleMaterial);
+  redPaddle.position.set(20, 0, 0);
+  bluePaddle.position.set(-20, 0, 0);
 
-  const bluePaddleGeometry = new THREE.BoxGeometry(0.5, 3, 3);
-  const bluePaddleMaterial = new THREE.MeshBasicMaterial({
-    color: 0x0000ff,
-    transparent: true,
-    opacity: 0.7,
+  // paddle 내부, 테두리
+  const bluePaddleBox = new THREE.Group();
+  const redPaddleBox = new THREE.Group();
+
+  const paddleBorder = (type) => {
+    return new THREE.BoxGeometry(
+      paddleWidth,
+      type === 'horizontal' ? paddleWidth - paddleWidth / 2 : paddleHeight,
+      type === 'horizontal' ? paddleDepth : paddleWidth - paddleWidth / 2
+    );
+  };
+
+  const paddleInner = (type) => {
+    return new THREE.BoxGeometry(
+      paddleWidth,
+      type === 'horizontal' ? paddleWidth / 2 : paddleHeight / 3,
+      type === 'horizontal' ? paddleDepth / 3 : paddleWidth / 2
+    );
+  };
```

Comment:
```suggestion
  const paddleBorder = (type) => {
    return new THREE.BoxGeometry(
      paddleWidth + 0.05,
      type === 'horizontal'
        ? paddleWidth - paddleWidth + 0.8 / 2
        : paddleHeight + 0.05,
      type === 'horizontal'
        ? paddleDepth + 0.05
        : paddleWidth - paddleWidth / 2 + 0.05
    );
  };

  const paddleInner = (type) => {
    return new THREE.BoxGeometry(
      paddleWidth + 0.05,
      type === 'horizontal' ? paddleWidth / 2 + 0.05 : paddleHeight / 3 + 0.05,
      type === 'horizontal' ? paddleDepth / 3 + 0.05 : paddleWidth / 2 + 0.05
    );
  };
```
회전했을 때 패들 색상 깨지는 문제가 안에 투명한 판이랑 보더랑 겹쳐서 그랬던 것 같아요! 물리학자 이원진 선생님이 고쳐주셨습니다. 이렇게 하니까 안깨지네요!

Reply:
이거 어떻게 해야할지 애매했는데 고쳐주셨군요 감사합니다 👍